src/config.js

```diff
@@ -23,7 +24,13 @@ class Config {
       store_id,
       retryDelay,
       retryJitter,
-      fetchMaxAttempts
+      fetchMaxAttempts,
+      throttleRequests,
+      throttleLimit,
+      throttleInterval,
+      throttleStrict,
+      httpKeepAlive,
+      httpKeepAliveInterval
```

Comment:
need to clean these up if we are removing keep alive from this PR

Reply:
I'll update it, thank you